README.md

```diff
@@ -46,11 +48,12 @@ Before deploying the bundle you may want to create a dedicated model for Tempo c
 juju add-model tempo
 ```
 
-### Monolithic deployment
-The bundle deploys Tempo in its [monolithic mode](https://grafana.com/docs/tempo/latest/setup/deployment/#monolithic-mode), which runs all required components in a single process and is the default mode of operation. Monolithic mode is the simplest way to deploy Grafana Tempo and is useful if you want to get started quickly or want to work with Grafana Tempo in a development environment.
+### Scaling monolithic deployment
+The bundle, by default, deploys Tempo in its [scaling monolithic mode](https://grafana.com/docs/tempo/latest/setup/deployment/#scaling-monolithic-mode), which runs all required roles within a single process. Scaling monolithic mode is the way to deploy Grafana Tempo with horizontal scale out by instantiating more than one `worker` process and is useful if you want to get started quickly or want to work with Grafana Tempo in a development environment.
 
 ```shell
-tox -e render-bundle -- bundle.yaml
+# render bundle with "edge" charms
+tox -e render-edge
 juju deploy ./bundle.yaml --trust
 ```
 
```

Comment:
at line 62, please add a note that what makes this mode 'monolithic' is that the worker application is configured with `role=all`.